Makefile

```diff
@@ -150,11 +151,14 @@ uuid-check:  ## list duplicated or empty uuids
 uuid-fix:
 	@scripts/fix_uuids.sh
 
+token-prefix-editor:
+	@scripts/token_editor.py
+
 gitflake8:
 	$(info "Checking style and syntax errors with flake8 linter...")
 	@flake8 $(shell git diff --name-only | grep ".py$$") --show-source
 
-can-i-push?: gitflake8 uuid-check test-docstrings test-robottelo
+can-i-push?: gitflake8 uuid-check test-docstrings test-robottelo token-prefix-editor
```

Comment:
Not sure if `token-prefix-editor` must be called here.

`can-i-push?` is not meant to change any file, it just check and warn.

if one have invalid token then

```bash
$make can-i-push?
ERROR: You have invalid token, please run make token-prefix-editor or fix it manually.
```